lighthouse-core/gather/driver/network-monitor.js

```diff
@@ -81,6 +82,23 @@ class NetworkMonitor {
     await session.sendCommand('Network.enable');
   }
 
+  /**
+   * @param {LH.Crdp.Target.AttachedToTargetEvent} event
+   */
+  async _onTargetAttachedLegacy(event) {
```

Comment:
i looked a this compared to my old solution:

![image (55)](https://user-images.githubusercontent.com/39191/171937284-2c68656a-46f8-4240-93f7-7ecaf5148e6f.png)
 
and see that you're still ensuring that networkmonitor calls network.enable for every new target. and my thing wouldn't have done that.

so.. 👍  !

Reply:
actually we don't need to do any of this. driver is already enabling the network domain for us.